app/components/SidebarLayout.tsx

```diff
@@ -163,6 +164,13 @@ export default function SidebarLayout({ children }: { children: ReactNode }) {
                 label="Locations"
                 href="/locations"
               />
+              <SideBarListItem
+                isCollapsed={isCollapsed}
+                isCurrentPage={segments.includes("create-crash-record")}
+                Icon={FaCirclePlus}
+                label="Create crash"
```

Comment:
I agree with Mike and Rose about combining the temporary crash records page with the crash records page, but if this does stay separate, I prefer "Create crash record" over "Create crash"

Reply:
Now that you mention it, I prefer that as well. That label forces the sidebar to be a lot wider, and i think we definitely need to shrink the sidebar font size, but since we are going to move this into the crashes page anyway I will leave it as-is for now 🙏